bin/utils/export_docs_generators.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Out of scope for this PR

Reply:
If you want to add this in this PR, for git-log clarity it should come in own separate commit.